lib/sunrise/sunrise.php

```diff
@@ -7,7 +7,7 @@
 	return;
 }
 
-$mu_plugin_dir = defined( 'WPMU_PLUGIN_DIR' ) ? constant( 'WPMU_PLUGIN_DIR' ) : constant( 'WP_CONTENT_DIR' ) . '/mu-plugins';
+$mu_plugin_dir = defined( 'WPVIP_MU_PLUGIN_DIR' ) ? constant( 'WPVIP_MU_PLUGIN_DIR' ) : constant( 'WP_CONTENT_DIR' ) . '/mu-plugins';
```

Comment:
This will not work. Sunrise is loaded before any mu plugins, including `000-vip-init.php` where the constant is first defined.

We'll want to define this constant earlier, in wp-config or something.

Reply:
I fixed it on https://github.com/Automattic/vip-go-mu-plugins/pull/5272/commits/a4cc270ac25cb7580713cd0c9fa686010a97e168 by using a relative path to it. WDYT?

Reply:
Mm, it doesn't feel super great but maybe is less flimsy than it first appears. If we were to define `WPVIP_MU_PLUGIN_DIR` earlier on though I do think that might be better.

Reply:
Yeah, i like that